cpp/util/etcd.cc

```diff
@@ -541,6 +541,10 @@ void EtcdClient::WatchInitialGetAllDone(WatchState* state, util::Status status,
 
   vector<WatchUpdate> updates;
   for (const auto& node : nodes) {
+    LOG_IF(INFO, etcd_index < node.modified_index_)
+        << "X-Etcd-Index (" << etcd_index
+        << ") smaller than node modifiedIndex (" << node.modified_index_
+        << ") for key \"" << node.key_ << "\"";
```

Comment:
I thought of adding a ```CHECK``` here, when I was reviewing the code that touches ```highest_index_seen_```, but figured this would just cause crashes when we could continue just fine.

Maybe I could make this a ```WARNING```, though? Considering it really shouldn't happen, and I was about to make the server crash instead of logging, maybe a flush to disk isn't the worst thing? :wink: